ci/taos/plugins-base/pr-audit-build-android.sh

```diff
@@ -168,6 +185,42 @@ function pr-audit-build-android-run-queue(){
         popd
         echo -e "[DEBUG] The current directory: $(pwd)."
 
+        # Start to build NNStreamer API for Android.
+        echo -e "[DEBUG] Starting gradle build for Android API."
+
+        # NNStreamer root directory for build.
+        export NNSTREAMER_ROOT=$(pwd)
+
+        # Make directory to build NNStreamer library.
+        sudo mkdir -p build_android_lib
```

Comment:
Is sudo really OK?
Is sudo really required?

Reply:
Changed permission of this script on aws (root > www) and removed sudo.
Now running build test in AWS.